lib/hooks/useCustomers.ts

```diff
@@ -10,9 +10,9 @@ interface CustomerResponse {
 
 function useCustomers(): CustomerResponse {
     // mock endpoint
-    const endpoint = 'http://localhost:3000/api/customers';
```

Comment:
Variables that aren't needed should be deleted, not commented out and left in commits.

Reply:
Fixed in 397993e